tests/aws/services/apigateway/test_apigateway_basic.snapshot.json

```diff
@@ -0,0 +1,17 @@
+{
+  "tests/aws/services/apigateway/test_apigateway_basic.py::TestAPIGateway::test_apigateway_with_step_function_integration[StartExecution]": {
+    "recorded-date": "07-09-2023, 22:40:26",
+    "recorded-content": {
+      "start_execution_response": {
+        "executionArn": "<executionArn:1>",
+        "startDate": "<startDate>"
+      }
+    }
+  },
+  "tests/aws/services/apigateway/test_apigateway_basic.py::TestAPIGateway::test_apigateway_with_step_function_integration[DeleteStateMachine]": {
+    "recorded-date": "07-09-2023, 22:40:38",
+    "recorded-content": {
+      "delete_state_machine_response": {}
```

Comment:
Is this snapshot intentionally empty?

Reply:
If the action is successful, the service sends back an HTTP 200 response with an empty HTTP body.